docs/Examples/watch-only-wallet.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const {HDPrivateKey, Network, Mnemonic} = require('bcoin');
+const {WalletClient} = require('bclient');
+
+(async () => {
+
+  // use well known test mnemonic
+  const phrase = [
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'about',
+  ].join(' ');
+
+  const network = Network.get('regtest');
+
+  const mnemonic = Mnemonic.fromPhrase(phrase);
+
+  // m'
+  const priv = HDPrivateKey.fromMnemonic(mnemonic);
+
+  // m'/44'
+  const bip44Key = priv.derive(44, true);
+
+  // m'/44'/0'
+  const bitcoinKey = bip44Key.derive(0, true);
+
+  // m'/44'/0'/0'
+  const accountKey = bitcoinKey.derive(0, true);
+
+  // account extended public key
+  // https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#Serialization_format
+  const xpub = accountKey.xpubkey(network.type);
```

Comment:
We need to be sure to turn the `accountKey` into a public key first

```
  // turn to public key
  const accountPubKey = accountKey.toPublic();
```

Reply:
@pinheadmz Good call on this: https://github.com/bcoin-org/bcoin/blob/master/lib/hd/private.js#L137-L139